src/opensynth/models/faraday/gaussian_mixture/fit_gmm.py

```diff
@@ -0,0 +1,160 @@
+# Copyright Contributors to the Opensynth-energy Project.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
We should reference the pycave package properly. For now, maybe add the following comment to the top of each file inside the `gaussian_mixture` directory:

Code based on source: Borchert, O. (2022). PyCave (Version 3.2.1) [Computer software] https://pycave.borchero.com/